kb/licensing/how-is-mbedtls-protected.md

```diff
@@ -4,7 +4,7 @@
 
 The copyright in the source code primarily protects Mbed TLS (formerly known as PolarSSL). We own all the copyright in the source code. For code that is contributed to us, we request the original author to share the copyright with us.
```

Comment:
"share the copyright" is a little vague. I suggest wording more like "to grant copyright to The Mbed TLS Contributors"

Reply:
... and, "We own all the copyright" is inaccurate - 3rdparty/everest is not under our copyright.

Reply:
... also the first sentence is a little unclear - not sure what it's adding.

Reply:
Actually, I think that's a hold-over from when Mbed TLS used a CLA, since that's what CLA's typically do